app/init/websocket_manager.ts

```diff
@@ -46,6 +46,10 @@ class WebsocketManager {
 
         AppState.addEventListener('change', this.onAppStateChange);
         NetInfo.addEventListener(this.onNetStateChange);
+
+        DeviceEventEmitter.addListener('windowFocusChanged', ({appState}: {appState: AppStateStatus}) => {
```

Comment:
I guess there is a case for this is emitted from Android only.  Perhaps, we can check the platform to avoid listening for this event of iOS; 1 event less to listen to ?

Reply:
Sure, not a problem